app/assets/javascripts/application.js

```diff
@@ -400,3 +400,27 @@ function setPowerState(item, status){
   power_actions.hide();
   $('[rel="twipsy"]').tooltip();
 }
+
+function disableButtonToggle(item) {
```

Comment:
I'd do `item = $(item)` at the beginning of the function and work with the benefits of jQuery later.